databricks/koalas/tests/test_dataframe.py

```diff
@@ -764,6 +766,23 @@ def check(op):
         check(lambda left, right: left.merge(right, left_on='lkey', right_on='rkey',
                                              suffixes=['_left', '_right']))
 
+        # Test Series on the right
+        check(lambda left, right: left.merge(right), right_ks, right_ps)
+        check(lambda left, right: left.merge(right, left_on='x', right_on='x'),
+              right_ks, right_ps)
+        check(lambda left, right: left.set_index('x').merge(right, left_index=True, right_on='x'),
+              right_ks, right_ps)
+
+        # Test join types with Series
+        for how in ['inner', 'left', 'right', 'outer']:
+            check(lambda left, right: left.merge(right, how=how), right_ks, right_ps)
+            check(lambda left, right: left.merge(right, left_on='x', right_on='x', how=how),
+                  right_ks, right_ps)
+
+        # suffix
+        check(lambda left, right: left.merge(right, suffixes=['_left', '_right'], how='outer',
+                                             left_index=True, right_index=True), right_ks, right_ps)
```

Comment:
Let's skip tests when pandas is older than 0.24. See the example with `LooseVersion` in `test_groupby.py`.